### PR TITLE
Update Selection API compat data for Chrome and Opera

### PR DESCRIPTION
**Pull request by hexrcs**
Selection API should be fully supported at least since Chrome 58/Opera 45.

### CODE REVIEW

File: api/Selection.json [hunk 1]
```diff
@@ -54,10 +54,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/addRange",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```
Comment (jpmedley) [inline api/Selection.json]: Should be `1`.  Leave the other flavors of Chrome as `true`.

File: api/Selection.json [hunk 2]
```diff
@@ -102,10 +102,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/anchorNode",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```
Comment (jpmedley) [inline api/Selection.json]: `1`

File: api/Selection.json [hunk 3]
```diff
@@ -150,10 +150,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/anchorOffset",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```
Comment (jpmedley) [inline api/Selection.json]: `1`

File: api/Selection.json [hunk 4]
```diff
@@ -777,10 +777,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/isCollapsed",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```
Comment (jpmedley) [inline api/Selection.json]: `1`

File: api/Selection.json [hunk 5]
```diff
@@ -1258,10 +1258,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Selection/toString",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```
Comment (jpmedley) [inline api/Selection.json]: `1`

Comment (hexrcs) [reply api/Selection.json]: Done.

**Pull request merged**